apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/InstancesTest.cs

```diff
@@ -66,7 +66,7 @@ void CreateInstanceNonAsciiDescription()
                 };
                 var insertOp = instancesClient.Insert(projectId, zone, instanceResource);
                 var completed = insertOp.PollUntilCompleted(metadataCallback: metadata => _output.WriteLine($"Called back; metadata name={metadata.Name}"));
-                Assert.Null(completed.Exception);
+                Assert.True(completed.Exception is null, completed.Exception.ToString());
```

Comment:
This should be `completed.Exception?.ToString()` otherwise if the exception *is* null, we'll throw when evaluating the second argument.

Reply:
Yes, duh, Fixed.